src/parser/lexer.rs

```diff
@@ -533,7 +533,7 @@ mod tests {
       DEFCAL RX(%theta) 0:
       \tPULSE 0 \"xy\" my_waveform()
       
-      DEFCAL MEASURE 0 %dest:
```

Comment:
[The spec](https://quil-lang.github.io/) seems to only use the "formal identifier" variant `%dest`. Is there a definition for this variant in this position?